Tests/SwiftIDEUtilsTest/ClassificationTests.swift

```diff
@@ -269,7 +269,7 @@ public class ClassificationTests: XCTestCase {
     assertClassification(
       #"#sourceLocation(file: "x", line: 1)"#,
       expected: [
-        ClassificationSpec(source: "#sourceLocation", kind: .poundDirective),
+        ClassificationSpec(source: "#sourceLocation", kind: .identifier),
```

Comment:
I think `#sourceLocation` being classified as `.identifier` makes more sense than `.ifConfigDirective` after the renaming. What do you think?

Reply:
I think that `keyword` might be the best classification, actually.

1. `#sourceLocation` is different than macros in that it modifies global state (namely the evaluation of `#file` and `#line` for all following lines)
2. `file` and `line` are also classified as keywords
3. It’s what we get by default, so it’s the least amount of work 😉